assets/js/renderer.js

```diff
@@ -355,8 +355,51 @@ $(document).ready(function () {
     // Center and reset node view
     resetZoomButton.click(() => nodeView.centerNode());
 
+    // Clear any saved splitter-related div widths from local storage when the app closes.
+    const clearDivs = () => {
+        if ('localStorage' in window && window['localStorage'] !== null) {
+            localStorage.removeItem('splitSizes');
+        }
+    }
+
+    window.addEventListener('beforeunload', function (event) {
+        clearDivs();
+    });
+
+    // Used in the split.js onDragEnd property below.
+    const saveResize = () => {
+        localStorage.setItem('splitSizes', JSON.stringify(instance.getSizes()));
+    }
+
+    // Instantiate the splitter.
+    const instance = Split(['#leftCol', '#tabbar'], {
+        sizes: [18, 76],
+        minSize: 200,
+        gutterSize: 5,
+        onDragEnd: saveResize
+    });
+
+    // Retrieve any saved resize settings from localStorage or else use our default setting.
+    function getDivs() {
```

Comment:
`getDivs` should be renamed to something more descriptive, like `restoreSplitterSizes`